src/main/java/gregtech/api/metatileentity/multiblock/MultiblockWithDisplayBase.java

```diff
@@ -537,11 +540,29 @@ public <T> T getCapability(Capability<T> capability, EnumFacing side) {
 
     @Override
     public boolean canVoidRecipeFluidOutputs() {
-        return voidingFluids;
+        return voidingFluids || fluidInfSink;
     }
 
     @Override
     public boolean canVoidRecipeItemOutputs() {
-        return voidingItems;
+        return voidingItems || itemInfSink;
+    }
+
+    public void checkStructurePattern() {
```

Comment:
these setters probably should be moved to the hatches, by overriding addToMultiBlock in MetaTileEntityMultiblockPart

Reply:
done